src/core/Notification/Notification.test.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { Notification } from './Notification';
+import { axeTest } from '../../utils/test';
+
+describe('Error view tests', () => {
+  const ErrorNotification = (
+    <Notification
+      status="error"
+      className="notification-test-class"
+      closeText="Close"
+      smallScreen
+      labelText="Lorem ipsum dolor sit"
+    >
+      Testcontent
+    </Notification>
+  );
+
+  it('should have status and smallScreen prop specific classNames', () => {
+    const { container } = render(ErrorNotification);
+    expect(container.firstChild).toHaveClass('fi-notification--error');
+    expect(container.firstChild).toHaveClass('fi-notification--small-screen');
+  });
+  it('should have given className as props', () => {
+    const { container } = render(ErrorNotification);
+    expect(container.firstChild).toHaveClass('notification-test-class');
+  });
+});
+describe('props', () => {
+  describe('children', () => {
+    const notificationWithElementChild = (
+      <Notification closeText="Close" labelText="Lorem ipsum dolor sit">
+        <div>Test notification</div>
+      </Notification>
+    );
+
+    it('has the given content', () => {
+      const { container } = render(notificationWithElementChild);
+      expect(container.firstChild).toHaveTextContent('Test notification');
+    });
+
+    it('should match snapshot', () => {
+      const { container } = render(notificationWithElementChild);
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  describe('classnames', () => {
```

Comment:
It would be better if the text would match the actual prop name, in this case `className`

Reply:
Fixed. is now classname